space-age.json

```diff
@@ -0,0 +1,54 @@
+{ "#": [],
+  "age on": {
+    "cases": [
+      {
+        "description": "Age on Earth",
+        "planet": "Earth",
+        "seconds": 1000000000,
+        "expected": 31.69
+      },
+      {
+        "description": "Age on Mercury",
+        "planet": "Mercury",
+        "seconds": 2213483568,
```

Comment:
for some reason this number is slightly different than what it is on https://github.com/exercism/xhaskell/blob/master/exercises/space-age/space-age_test.hs - 2213483568 as opposed to 2134835688 - this causes the answer to be 291.23 instead of 280.88 - one of the two should be changed